lib/ansible/modules/cloud/vmware/vmware_host_ntp.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_host_ntp
+short_description: Manage NTP configurations about an ESXi host
+description:
+- This module can be used to manage NTP configuration information about an ESXi host.
+- User can specify an ESXi hostname or Cluster name. In case of cluster name, all ESXi hosts are updated.
+version_added: '2.5'
+author:
+- Abhijeet Kasurde (@akasurde)
+notes:
+- Tested on vSphere 6.5
+requirements:
+- python >= 2.6
+- PyVmomi
+options:
+  cluster_name:
+    description:
+    - Name of the cluster.
+    - NTP settings are applied to every ESXi host system in given cluster.
+    - If C(esxi_hostname) is not given, this parameter is required.
+  esxi_hostname:
+    description:
+    - ESXi hostname.
+    - NTP settings are applied to this ESXi host system.
+    - If C(cluster_name) is not given, this parameter is required.
+  ntp_server:
+    description:
+    - "IP or FQDN of NTP server/s."
```

Comment:
Probably good to add how you configure multiple servers (as a list, or specify `For multiple servers, please look at the examples`)

Very minor: Maybe `ntp_servers` (plural) is a better name? Just to indicate that more can be specified?